pkg/operator/starter.go

```diff
@@ -131,7 +140,7 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 			"v4.1.0/kube-apiserver/localhost-recovery-client-crb.yaml",
 			"v4.1.0/kube-apiserver/localhost-recovery-sa.yaml",
 			"v4.1.0/kube-apiserver/localhost-recovery-token.yaml",
-			"v4.1.0/kube-apiserver/audit-policies-cm.yaml",
+			"pkg/operator/apiserver/audit/manifests/audit-policies-cm.yaml",
```

Comment:
how do we do this in other operators? Seeing audit here is a little strange.

Reply:
https://github.com/openshift/cluster-openshift-apiserver-operator/blob/master/pkg/operator/starter.go#L193

Reply:
@EmilyM1 try `libgoaudit.AuditPoliciesConfigMapFileName` here